### PR TITLE
CHAD-4014, WWST-6257 Add Window Shade Preset support to Zigbee Window Shades, Update preference text

### PR DESCRIPTION
**Pull request by dkirker**
We've decided that we will add preset position support for these devices.

### CODE REVIEW

**Comment by dkirker** [comment]
I still need to run it and test it.

**Review by tpmanley — DISMISSED** [review]

File: devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy [hunk 1]
```diff
@@ -184,6 +192,10 @@ def pause() {
 	zigbee.command(CLUSTER_WINDOW_COVERING, COMMAND_PAUSE)
 }
 
+def presetPosition() {
+    setLevel(preset ?: 50)
```
Comment (greens) [inline devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy:196]: it looks like zwave window shade uses `setLevel(preset ?: state.preset ?: 50)` idk which is more correct

Comment (dkirker) [reply devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy:196]: @greens Yeah, I saw that. I'm not sure that I completely follow the logic. I'm not sure if preferences are accessible through state like that? They could be. I've always seen just directly using the name as a variable, or settings.prefName. I'll check this out as I am testing it.

**Comment by tpmanley** [comment]
@dkirker I suggest breaking this into two PRs. One to add `presetPosition` to DTHs that don't have it (CHAD-4014) and a second to fix the text on the remaining DTHs (WWST-6257).

**Comment by dkirker** [comment]
@tpmanley I broke out the WWST-6257 stuff from the device handlers that weren't covered by CHAD-4014 and will create another PR for the other two device handlers.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**